### PR TITLE
updated the html,css to keep navbar sticky

### PR DESCRIPTION
**Pull request by namrata18s**

### CODE REVIEW

File: rank.html [hunk 1]
```diff
@@ -53,7 +53,7 @@
     </div>
     <div id="mobile-view">
 
-        <nav id="myNav" class="navbar navbar-expand-lg">
+        <nav id="myNav" class="navbar navbar-expand-lg fixed fixed-top">
```
Comment (namrata18s) [inline rank.html:56]: my conflict on line 56 can be removed.

File: css/rank.css [hunk 2]
```diff
@@ -15,4 +16,39 @@
     -webkit-animation: spin 2s linear infinite;
     /* Safari */
     animation: spin 2s linear infinite;
-}
```
Comment (MaathavanJkr) [inline css/rank.css:18]: This should not be removed ig

**Comment by MaathavanJkr** [comment]
Now the navbar became sticky but the table headers aren't sticky.

**Comment by namrata18s** [comment]
yes actually it may be scrolling behind navbar.

**Comment by MaathavanJkr** [comment]
@namrata18s hello, if you have any difficulties I'm here to help. ;).

**Comment by namrata18s** [comment]
Yes @MaathavanJkr actually jt was taking a long time for me to resolve it....

**Comment by MaathavanJkr** [comment]
> Yes @MaathavanJkr actually jt was taking a long time for me to resolve it....

Tip: I think the navbar has a fixed height. So Make the table headers sticky margin top as the height of the navbar. It should work ig. 🤔

**Comment by MaathavanJkr** [comment]
And better discard your older changes and fetch again to resolve the conflict first. Then make the new changes. :)

**Comment by MaathavanJkr** [comment]
any improvement?

**Pull request closed without merging**